<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>locationLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="xpLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>xpLabel </string>
    </property>
   </widget>
   <widget class="QLabel" name="healthLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>healthLabel </string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkInventoryButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveGameButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadGameButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load game</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="tlo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-30</y>
      <width>831</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>801</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="history">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>History about location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trash">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newGame">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="plainText">
     <string>---OBJECTIVES----

- Enter Volcanic Mountains

- Defeat The Amber Lord</string>
    </property>
   </widget>
   <zorder>tlo</zorder>
   <zorder>moveButton</zorder>
   <zorder>checkInventoryButton</zorder>
   <zorder>saveGameButton</zorder>
   <zorder>loadGameButton</zorder>
   <zorder>TextEdit</zorder>
   <zorder>healthLabel</zorder>
   <zorder>xpLabel</zorder>
   <zorder>locationLabel</zorder>
   <zorder>history</zorder>
   <zorder>trash</zorder>
   <zorder>quitButton</zorder>
   <zorder>newGame</zorder>
   <zorder>plainTextEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
